### ISSUE TITLE
Question: it is neccessary to output console.error(error, messages.failedToFix(...)) in catch of autoFixFunc ?

### ISSUE THREAD

**Issue by voskresla** [AndyOGo/stylelint-declaration-strict-value#276]
With this lines we get additional console error besides the report. When we pass `throw null` we get this `null` before `default error`

https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/fc662943ede256727f4a5350f29cc27941073e84/src/index.ts#L442-L445

```
'scale-unlimited/declaration-strict-value': ['z-index', { 
    ignoreFunctions: false, 
    autoFixFunc: () => { throw null } 
}],
```

![image](https://user-images.githubusercontent.com/10416628/195576667-a18ee29d-fb13-4c74-88a2-a4936b66613b.png)

**Comment by AndyOGo** [comment]
`() => { throw null }` doesn't automatically fix something, right?

I don't know what you want to archive, but that config is not correct.

Maybe you just need to the `disableFix` option.

Otherwise please read the documentation about autofixing
https://github.com/AndyOGo/stylelint-declaration-strict-value#autofix-support

**Comment by AndyOGo** [comment]
@voskresla 
Thanks a lot for the issue.

I removed it an released [`v1.9.1`](https://github.com/AndyOGo/stylelint-declaration-strict-value/compare/v1.9.0...v1.9.1)
